variables.tf

```diff
@@ -9,6 +9,12 @@ variable "base_domain" {
   default     = null
 }
 
+variable "domain_id" {
```

Comment:
In the EKS module, we don't have this variable but we use a datasource to find the domain id from the `base_domain` variable. I guess the reason is that the domain is often not managed in the same terraform state as the devops-stack. I'm not sure if it's better to do the same in this module or if there's a reason to do it differently... :shrug: ?

Reply:
I think we can do it similarly, it's just that I probably had some issues with it in the beggining and then never came back to it. I'll add a data source and test it.